lib/matplotlib/__init__.py

```diff
@@ -1839,7 +1839,7 @@ def inner(ax, *args, **kwargs):
 
     return param
 
-_log.info('matplotlib version %s', __version__)
-_log.info('interactive is %s', is_interactive())
-_log.info('platform is %s', sys.platform)
+_log.debug('matplotlib version %s', __version__)
```

Comment:
Don’t agree w these three. If we ask a user to debug their code af the info level we will want this info won’t we?

Reply:
Then just ask the user to report the DEBUG level log?...

In my view:
INFO = stuff that may be of general interest to a slightly technically savvy end user, if anything appears to be wrong.
DEBUG = stuff that we want to see in bug reports because we have no idea how the reporter's system is set up.

Reply:
(As above, my main use case personally is that I just always have a logger on at INFO, and for most modules it's nice to have that around, and I can live with a little extra noise, but having it on literally every mpl import is a bit too noisy.  Yes, I could also change my setup but meh.)